splunk_connect_for_snmp/inventory/loader.py

```diff
@@ -130,6 +133,19 @@ def load():
                         {"$set": ir.asdict()},
                         upsert=True,
                     )
+                    profiles = source_record["profiles"].split(";")
+                    profile = None
+                    if profiles:
+                        profiles = list(
```

Comment:
that can be simply: 
`[p for p in config_profiles if p.get("condition", {}).get("type") == "walk"]`